src/commcare_cloud/commands/terraform/terraform.py

```diff
@@ -87,7 +92,7 @@ def run(self, args, unknown_args):
                 else ''
             )
 
-            with open(os.path.join(run_dir, 'secrets.auto.tfvars'), 'w') as f:
+            with open_python_dependent(os.path.join(run_dir, 'secrets.auto.tfvars')) as f:
                 print('rds_password = {}'.format(json.dumps(rds_password)), file=f)
```

Comment:
This doesn't need to use `open_python_dependent`, does it? The template string is a literal, so text. Please tell me `environment.get_secret()` returns text (if not, we should make it so).

Reply:
Ah I believe you are right. I saw `json.dumps` and didn't think enough about the rest of this.

Just to talk through it, (on Python 2) as long as `rds_password` contains ASCII characters only, `json.dumps` will return a bytes string which will then be converted to unicode due to the string literal formatting.

Reply:
> as long as `rds_password` contains ASCII characters only, `json.dumps` will return a bytes string

`json.dumps` will always return ASCII bytes on Python 2. Non-ASCII characters are escaped.

> which will then be converted to unicode due to the string literal formatting.

correct.